lib/parser.js

```diff
@@ -729,6 +729,15 @@ const deprecatedArgs = [
     deprecatedFor: '--default-capabilities',
     help: '[DEPRECATED] - (IOS-only) if set, the iOS system log will be written to the console',
     nargs: 0,
+  }],
+
+  [['--enable-heapdump'], {
+    defaultValue: false,
+    dest: 'isHeapdumpEnabled',
```

Comment:
The naming scheme throughout does not include "is". `heapdumpEnabled` would be more in line.